tests/OnlineSales.Tests/BaseTest.cs

```diff
@@ -91,6 +91,16 @@ protected async Task<HttpResponseMessage> GetTest(string url, HttpStatusCode exp
         }
     }
 
+    protected async Task<T?> GetTestRawContentSerialize<T>(string url, HttpStatusCode expectedCode = HttpStatusCode.OK, string authToken = "Success")

```

Comment:
@Exmirai please avoid to change BaseTest.cs file if you are not sure about reusability of the particular method you create. I can see that GetTestRawContentSerialize is only used once, please move that logic to the place where you need it and only add it to the BaseTest.cs if you see multiple application of that method across different inherited classes.